Tests/Editor/JsonParsingTests.cs

```diff
@@ -471,5 +471,32 @@ public void ParseGarbage()
             gltf = JsonParser.ParseJson(@"garbage");
             Assert.IsNull(gltf);
         }
+        
+        [Test]
+        public void A28Guid()
```

Comment:
Test passes when run locally, using instructions on [this thread](https://github.com/atteneder/glTFast/issues/440). Also, one of the submodules in [gltfasttest](https://github.com/atteneder/glTFastTest) is quite large (1.7GB). Don't think it's worth running on CI, but let me know what you guys think 🤔